### PR TITLE
Add Install Informations for windows

### PR DESCRIPTION
**Pull request by shinobu**
This pull requests adds a few lines to help windows users to potentially build ontowiki from source.

You can check through it @white-gecko

### CODE REVIEW

**Comment by white-gecko** [comment]
Which source code has to be compiled for installing OntoWiki? Maybe you test some kind of a WAMP stack.

**Comment by shinobu** [comment]
i guess i should reword it to building the environment? better ideas to call it?
